src/client/Dashboard.tsx

```diff
@@ -27,6 +25,17 @@ import { DeploymentTable } from "./DeploymentTable";
 
 const REFRESH_INTERVAL = 30000;
 class Dashboard<Props> extends React.Component<Props, IDashboardState> {
+  public state: IDashboardState = {
+    authors: {},
+    deployments: [],
+    filteredDeployments: [],
+    prs: {},
+    rowLimit: Number.parseInt(
+      new URLSearchParams(location.search).get("limit") ?? "50", // default to 50 rows
+      10
+    )
```

Comment:
just being paranoid, what happen where we have `limit=-10`

Reply:
No cleaning is currently done for `limit`. The effect that would have is having no deployments displayed -- we just use a `.slice` based on the limit (i.e. `deploymentsToShow = deployments.slice(0, this.state.rowLimit)`, which would be an empty list)

Reply:
ok we are safe then. thanks